plugin.audio.rco/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.rco" name="Het Koninklijk Concertgebouworkest" version="1.0.1" provider-name="bosel">
+  <requires>
+      <import addon="xbmc.python" version="2.14.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="rco.py">
+      <provides>audio</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+      <summary lang="en">The Royal Concertgebouw Orchestra</summary>
+      <summary lang="nl">Het Koninklijk Concertgebouworkest</summary>
+      <description lang="en">The Royal Concertgebouw Orchestra is a symphony orchestra that gives orchestral performances of the highest calibre in the world’s leading concert halls under the direction of the very best conductors. The activities it carries out in Amsterdam form the basis of its role as the Netherlands’ ambassador for international excellence. The Royal Concertgebouw Orchestra offers audiences emotional and intellectual enrichment, generating involvement and active loyalty.</description>
+      <description lang="nl">Het Koninklijk Concertgebouworkest is het symfonieorkest dat wereldwijd op de belangrijkste podia, onder leiding van de beste dirigenten en op het allerhoogste niveau uitvoeringen geeft van orkestmuziek. Zijn activiteiten in Amsterdam vormen de basis voor zijn rol als Nederlands ambassadeur van internationale excellentie. Het Concertgebouworkest biedt zijn publiek emotionele en intellectuele verrijking en verleidt tot betrokkenheid en actieve loyaliteit.</description>
+      <platform>all</platform>
+      <language>nl en</language>
```

Comment:
language tag refers to media content and not to translations. Is the content (audio) this addon provides in english and dutch? If not please change to the correct value (probably just nl).